etna/transforms/missing_values/resample.py

```diff
@@ -96,6 +96,7 @@ def transform(self, df: pd.DataFrame) -> pd.DataFrame:
         :
             result dataframe
         """
+        df = df.apply(pd.to_numeric)
```

Comment:
You also need to add test to check that you have fixed sth. You can use setup from the issue as base